docs/admin/Metadata/index.rst

```diff
@@ -0,0 +1,115 @@
+.. admin_Metadata_index:
+
+.. _`audittrail` : ../Logging/index.hmtl
+.. _`Wet open overheid` : https://wetten.overheid.nl/BWBR0045754/
+.. _`Woo-index`: https://open.overheid.nl/
+.. _`DiWoo` : https://standaarden.overheid.nl/diwoo/metadata
+.. _`Woo informatiecategorieën` : https://standaarden.overheid.nl/tooi/waardelijsten/work?work_uri=https%3A%2F%2Fidentifier.overheid.nl%2Ftooi%2Fset%2Fscw_woo_informatiecategorieen
+.. _`Thema-indeling voor Officiële Publicaties (TOP-lijst)` : https://standaarden.overheid.nl/tooi/waardelijsten/work?work_uri=https%3A%2F%2Fidentifier.overheid.nl%2Ftooi%2Fset%2Fscw_toplijst
+.. _`audittrail` : ../Logging/index.hmtl
```

Comment:
we tend to put the link targets at the bottom of a file usually to increase readability of the code

additionally, you don't need to wrap the aliases in backticks for the link definitions :)

```suggestion
.. _Wet open overheid: https://wetten.overheid.nl/BWBR0045754/
.. _Woo-index: https://open.overheid.nl/
.. _DiWoo: https://standaarden.overheid.nl/diwoo/metadata
.. _Woo informatiecategorieën: https://standaarden.overheid.nl/tooi/waardelijsten/work?work_uri=https%3A%2F%2Fidentifier.overheid.nl%2Ftooi%2Fset%2Fscw_woo_informatiecategorieen
.. _Thema-indeling voor Officiële Publicaties (TOP-lijst): https://standaarden.overheid.nl/tooi/waardelijsten/work?work_uri=https%3A%2F%2Fidentifier.overheid.nl%2Ftooi%2Fset%2Fscw_toplijst
```

This also removes the internal links, for those we can use the `ref` syntax, e.g.:

```rst
:ref:`admin_logging_index`
```